gitlab-cng-17.1.yaml

```diff
@@ -1,11 +1,7 @@
-# source is gitlab so we can't use github updates to get expected commit
-# let's still auto create the PR, it will fail as expected commit will be wrong
-# however it will be easy to fix
-#nolint:git-checkout-must-use-github-updates
 package:
-  name: gitlab-cng
+  name: gitlab-cng-17.1
   version: 17.1.1
-  epoch: 1
+  epoch: 0
   description: Cloud Native container images per component of GitLab
   copyright:
     - license: MIT
```

Comment:
shouldn't we add provides here for gitlab-cng?

Reply:
Added